go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/dovics/session-role-manager
```

Comment:
It should be github.com/dovics/session-role-manager/v2. 
What do you think? @dovics

Reply:
sorry, I forget fix it